lib/resources/example_json/r4_examples_charge_item.rb

```diff
@@ -117,6 +153,15 @@ module Resources
       'code': {
         'coding': [
           {
+            'extension': [
+              {
+                'valueCoding': {
+                  'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/14002',
+                  'code': '615214'
```

Comment:
Is 615214 actually returning as the bill code schedule value here? This seems like it should be value 656909 which is the generic CDM_SCHED one.

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/629/commits/8a612215cd46509787f4a54f29c6120975f44c27#diff-5cd46ad5f0af8a24217f67b656e77ce3624d6cefec3a18eddf7adc12bd47d35fR160

Reply:
+1